src/Theme.elm

```diff
@@ -98,6 +98,30 @@ pureWhite =
     hex "ffffff"
 
 
+
+-- Breakpoints
+
+
+maxMobile : Float
+maxMobile =
+    576
+
+
+withMediaMobile : List Style -> Style
```

Comment:
I think we should find/replace with some other names, but otherwise nice to have these here

Reply:
What names? I can change it.